src/Mapper/BaseMapper.php

```diff
@@ -21,7 +21,7 @@
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-abstract class BaseMapper
+abstract class BaseMapper implements MapperInterface
```

Comment:
Does it make sense at all to have this abstract... previously all method were abstract, and now it has none.

To me , an abstract class makes sense when it have some logic and some abstract method, but this class does not have any of that (previously or after this pr)

Reply:
We can deprecate it indeed.